listenbrainz/webserver/views/profile.py

```diff
@@ -168,6 +185,25 @@ def export_data():
         return render_template("user/export.html", user=current_user)
 
 
+@profile_bp.route("/export-feedback", methods=["POST"])
+@login_required
+def export_feedback():
+    """ Exporting the feedback data to json """
+    filename = current_user.musicbrainz_id + "_lb_feedback-" + datetime.today().strftime('%Y-%m-%d') + ".json"
+
+    # Build a generator that streams the json response. We never load all
+    # feedback into memory at once, and we can start serving the response
+    # immediately.
+    feedback = fetch_feedback(current_user.id)
+    output = stream_json_array(_feedback_to_api(fb=fb) for fb in feedback)
+
+    response = Response(stream_with_context(output))
```

Comment:
this code is duplicated from the export listens function right? We should extract it into a seperate function.